### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by SaloCreative**
- Package updates
- Adding CI config 
- Making map more configurable
- Removing lakes

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # interactive-map
 
-DEMO here [https://salocreative.github.io/react-interactive-map](https://salocreative.github.io/react-interactive-map)
+DEMO here [https://salo-reative.github.io/react-interactive-map](https://salo-creative.github.io/react-interactive-map)
```
Comment (fetimo) [inline README.md]: Typo

File: README.md [hunk 2]
```diff
@@ -40,4 +40,26 @@ Implement as follows
...
+
+Then run the demo command to run the initial unit tests and boot up the development server
+```bash
+yarn demo
+```
+
+
+## Publishing
+
+// add this when finalised
```
Comment (fetimo) [inline README.md]: Did you mean to leave this comment in?

File: ci/codefresh.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    git: salocreative
+  test:
+    image: node:10
+    working_directory: ${{main_clone}}
+    stage: "test"
+    commands:
+      - yarn
+      - yarn test
+  build:
+    image: node:10
```
Comment (fetimo) [inline ci/codefresh.yml]: Maybe 12 as it's a new build?

File: package.json [hunk 4]
```diff
@@ -53,75 +60,88 @@
...
   },
   "peerDependencies": {
-    "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "styled-components": "^4.2.0"
   },
   "dependencies": {
```
Comment (fetimo) [inline package.json]: Think some of these should be in dev/peer deps

File: README.md (hunk 2 above)
Comment (SaloCreative) [reply README.md]: Yeah as not really sure what to do about the release process still haha

File: ci/codefresh.yml (hunk 3 above)
Comment (SaloCreative) [reply ci/codefresh.yml]: Good call

File: README.md (hunk 1 above)
Comment (SaloCreative) [reply README.md]: Good spot

**Pull request merged**